subcommittees/README.md

```diff
@@ -0,0 +1,27 @@
+#Software Carpentry Foundation Subcommittees and Task Forces
+
+## Purpose and Rationale
+Software Carpentry is volunteer-powered organization that is rapidly growing. One of the greatest things about Software Carpentry is the huge diversity of people that have come together to collaborate, teach, and share ideas and resources. Your contributions to the community have made Software Carpentry a huge success!
+
+To meet the needs of our growing community, the 2015 Steering Committee Launched the first Subcommittee program to allow groups of members to organize and develop resources and activities that support our members. 
+
+The 2016 Steering Committee would like to encourage and invite the community to propose new initiatives in the form of subcommittees (a standing group for ongoing activities) and task forces (an ad hoc group focused on a finite task). Read on to learn more about existing initiatives and for information about how to propose a new initiatives that will shape our community.   
+
+## Current Subcommittees and Task Forces
+For information on existing and past subcommittees, please visit: https://github.com/swcarpentry/board/blob/master/subcommittees/subcommittees.md
+
+## How to Propose a New Subcommittee or Task Force
+Do you have an idea for an activity or service that would make Software Carpentry even more awesome to participate in for its instructors or students? Is there a project, tool or document that would benefit you and your community?
+
+We are very excited to hear and support your ideas! However, we do need to read and review all proposals.
+
+**To propose a new subcommittee or task force, please read the [information here](https://github.com/swcarpentry/board/blob/subc-tf/subcommittees/proposal_instructions.md)**. The key elements of a proposal that the Steering Committee will be looking for (and which we will help you with!) are:
```

Comment:
I'd rather have a relative target here too (e.g. `proposal_instructions.md`).  The current target is using the PR branch, which will probably go away after this PR lands.  Same comment for other intra-repo links.